### DISCUSSION TITLE
Git Classroom Adding Test Cases causes 500 error

### DISCUSSION THREAD

**Discussion by nashnj13** [community/community#116600] [GitHub Education]
### Hi everyone,

I have come across an issue (I am opted-in with the new fork changes) and when I am trying to create an assignment with test cases, hitting the complete assignment button sends me to a 500 error page and I lose my assignment.

This does not happen when I am not using test cases or create the assignment without them.

**Comment by samus-aran** [comment]
Hi @nashnj13 👋 

Are you still experiencing this problem?

**Reply by nashnj13** [reply]
No, this issue appears to have been fixed.
